docs/contributing.rst

```diff
@@ -25,6 +25,35 @@ below.
 .. note:: If you do not wish to use GitHub, please send patches to
           infrastructure@lists.fedoraproject.org.
 
+Development Environment
+=======================
+Vagrant allows contributors to get quickly up and running with a Securitas development environment by
+automatically configuring a virtual machine. To get started, first install the Vagrant and Virtualization 
+packages needed, and start the libvirt service::
+
+    $ sudo dnf install ansible libvirt vagrant-libvirt vagrant-sshfs vagrant-hostmanager
+    $ sudo systemctl enable libvirtd
+    $ sudo systemctl start libvirtd
+
+Check out the code and run ``vagrant up``::
+
+    $ git clone https://github.com/fedora-infra/securitas
+    $ cd securitas
+    $ vagrant up
+
+Next, SSH into your newly provisioned development environment, and start the securitas web application:
+
+    $ vagrant ssh
+    $ cd /vagrant/
+    $ poetry run flask run --host=0.0.0.0
```

Comment:
You could instruct devs to just use `securitas-devel` here, no?

Reply:
I would go one further and say that we should probably just start securitas as part of the vagrant box coming up. If we want to go with this, for now, I can add that to a separate PR.

Reply:
We can, but in any case we should tell devs how to run the server, in case they want to stop it to fix something and restart it later on.